tools/iothub-explorer/iothub-explorer.js

```diff
@@ -270,6 +271,21 @@ else if (command === 'receive') {
     });
   });
 }
+else if (command === 'sas-token') {
+  if (!arg1) inputError('No device ID given');
+  var registry = connString ? Registry.fromConnectionString(connString) : Registry.fromSharedAccessSignature(sas.toString());
+  registry.get(arg1, function (err, device) {
+    if (err)
+      serviceError(err);
+    else {
+      var key = device.authentication.SymmetricKey.primaryKey;
+      var expiry = arg2 ? Math.ceil((Date.now() / 1000) + (3600 * arg2)) : anHourFromNow();
+      console.log('SAS Token for device ' + arg1 + ' with expiry ' + (arg2 ? arg2 : 'one') + ' hour(s) from now:');
```

Comment:
Print only the token if user specified --raw. Otherwise, print this message in {green}.